blaze/compute/pandas.py

```diff
@@ -165,6 +165,21 @@ def compute_up(expr, data, **kwargs):
     return compute_up(expr._child.distinct().count(), data, **kwargs)
 
 
+@dispatch(Repeat, Series)
+def compute_up(expr, data, **kwargs):
+    return expr.lhs.str.repeat(data)
```

Comment:
Okay, I will leave these as is; however, I still do not see the methods on my `InteractiveSymbol`.

```python
In [8]: ds = bz.Data(pd.Series(('a', 'b', 'c')))

In [9]: 'concat' in dir(ds)
Out[9]: False

In [10]: ds.data + ds.data
Out[10]: 
0    aa
1    bb
2    cc
dtype: object
```